test/script/tskdbdiff.py

```diff
@@ -51,6 +54,12 @@ def __init__(self, output_db, gold_db, output_dir=None, gold_bb_dump=None, gold_
         self._bb_dump = ""
         self._dump = ""
         self.verbose = verbose
+        self.isMultiUser = isMultiUser
+        self.pgSettings = pgSettings
+
+        if self.isMultiUser and not self.pgSettings:
+            print("Missing Multi-user Settings data.")
+            sys.exit(1)
```

Comment:
I think we can make this error message a little more specific, since we know it is the PostgreSQL connection settings that are missing.